nilearn/decoding/space_net.py

```diff
@@ -777,11 +778,12 @@ def fit(self, X, y):
         n_samples, _ = X.shape
         y = np.array(y).copy()
         l1_ratios = self.l1_ratios
-        if isinstance(l1_ratios, numbers.Number):
+        if not isinstance(l1_ratios, collections.abc.Iterable):
             l1_ratios = [l1_ratios]
         alphas = self.alphas
-        if isinstance(alphas, numbers.Number):
-            alphas = [alphas]
+        if alphas is not None:
```

Comment:
Is this case currently tested ?

Reply:
It is tested implicitly: 
```
>>> import numbers
>>> isinstance(None, numbers.Number)
False
>>> 
```

Reply:
Sorry, I meant: is it covered by unit tests ?

Reply:
Ah, I don't know. It would be nice to add a test if not. That said this is a straightforward change. Just to shut up LGTM.com.